src/main/java/dk/bec/bookanything/mapper/WeekDay.java

```diff
@@ -0,0 +1,30 @@
+package dk.bec.bookanything.mapper;
+
+public enum WeekDay {
+    MONDAY(1,"Monday"),
+    TUESDAY(2,"Tuesday"),
+    WEDNESDAY(3,"Wednesday"),
+    THURSDAY(4,"Thursday"),
+    FRIDAY(5,"Friday"),
+    SATURDAY(6,"Saturday"),
+    SUNDAY(7,"Sunday");
+
+    private final int dayNumber;
+    private final String dayName;
+    private final static String DEFINITION_NOT_FOUND_MSG = "Definition not available";
+
+    WeekDay(int dayNumber,String dayName){
+        this.dayNumber = dayNumber;
+        this.dayName = dayName;
+    }
+
+    public static String getDayNameFromDayNumber(int dayNumber)
+    {
+        for(WeekDay weekDay: WeekDay.values())
```

Comment:
This logic is a bit weird, why scan enum values every time when you are sure that Monday is number 1, Friday is 5 etc.. ALWAYS. 
This whole structure looks more like a Map than Enum.